packages/js/src/ui/components/primitives/Popover/PopoverContent.tsx

```diff
@@ -6,7 +6,7 @@ import { Root } from '../../elements';
 import { usePopover } from './PopoverRoot';
 
 export const popoverContentVariants = () =>
-  'nt-flex-col nt-gap-4 nt-h-[600px] nt-min-w-[400px] nt-rounded-xl nt-bg-background nt-shadow-[0_5px_15px_0_rgba(122,133,153,0.25)] nt-z-10 nt-cursor-default nt-flex nt-flex-col nt-overflow-hidden';
+  'nt-flex-col nt-h-[600px] nt-w-[400px] nt-rounded-xl nt-bg-background nt-shadow-[0_5px_15px_0_rgba(122,133,153,0.25)] nt-z-10 nt-cursor-default nt-flex nt-flex-col nt-overflow-hidden';
```

Comment:
From one side the width should be restricted to 400px, from another angle it should be dictated by the content of the notification list item. I'm not sure 🤷‍♂️

Reply:
I'd say content of list

Reply:
yes, I thought the same, but without the restrictions on the Inbox Popover width, the tabs component will enlarge its width :/

Reply:
I feel we should have width constraints on the Popover and the let the notification content inherit the parents width

Reply:
Letting content dictate the popover's width can make the popover grow out of bounds of the viewport. ideally content should be  responsive enough to be able render in all sizes of screen and also as full page solutions